docs/mkdocs.yml

```diff
@@ -43,10 +44,9 @@ nav:
   - Release notes: release-notes.md
   - Operations:
     - Server deployment:
-      - Machine setup:
-         - Overview: machine-setup.md
-         - Cloud Deployment: azure.md
-         - Self-Hosting: ias.md
+        - Machine Setup: machine-setup.md
+        - Cloud Deployment: azure.md
```

Comment:
Can we have this the first in the line as it's going to be the most common way to deploy.

Reply:
The machine-setup page provides an overview of azure deployment and IAS. So, if we change the order, it would affect the flow. I recommend to keep the order as is.

Reply:
Whilst IAS isn't deprecated by Intel, it is no longer the default remote attestation protocol, which is now DCAP, and that is what's available on Azure. So the problem with the current page layout is that IAS is given Operations front page whilst the more common and default Azure/DCAP has to accessed via another click.

We definitely need to fix this, but I'm happy for you to restructure as part as of your general rewrite of the docsite. I appreciate the original PR was raised to fix the screenshots and azure docs. I'll approve to get this merged.

Reply:
I will add a ticket to the backlog to fix this.